src/components/internal/Backlog.component.tsx

```diff
@@ -156,7 +174,7 @@ export class BacklogComponent extends Component<Props, State> {
                     task={task}
                     numActionColumns={1}
                     showStatus={true}
-                    isSelectingOwner={task.id === this.props.selectOwnerTaskId}
+                    isSelectingOwner={this.props.selectOwnerTaskId === task.id}
```

Comment:
```suggestion
                    isSelectingOwner={task.id === this.props.selectOwnerTaskId}
```
prefer not to touch it if there is no change